src/inputs/tag-pattern.ts

```
@@ -0,0 +1,33 @@
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+import { cache } from "#utils/cache";
+import { getInput } from "@actions/core";
+
+export function tagPattern(): string | undefined {
+  return cache("tag-pattern", () => {
+    const value = getInput("tag-pattern");
+    return value || undefined;
```

ESLint

[error] Unexpected use of undefined.

(no-undefined)